ccViewer/ccViewer/ViewControllerImage.swift

```diff
@@ -8,11 +8,33 @@
 
 import UIKit
 import RemoteCloud
+import MapKit
+
+/* Placeholder - this one does not work as UIImage strips most meta data. */
+extension UIImage {
+
+    func getExifData() -> CFDictionary? {
+        var exifData: CFDictionary? = nil
+        if let data = self.jpegData(compressionQuality: 1.0) {
+            data.withUnsafeBytes {
+                let bytes = $0.baseAddress?.assumingMemoryBound(to: UInt8.self)
+                if let cfData = CFDataCreate(kCFAllocatorDefault, bytes, data.count),
+                    let source = CGImageSourceCreateWithData(cfData, nil) {
```

Comment:
Encoding the entire image just to get the exif data from it? That's pretty yucky. Can we go find the original file instead?

If we have to do this, at least cast the `data` to a CFData instead of copying the whole thing.